MSAL/src/MSALPublicClientApplication.m

```diff
@@ -1132,6 +1131,8 @@ - (void)acquireTokenWithParameters:(MSALInteractiveTokenParameters *)parameters
         return;
     }
     
+    [msidParams fillWithAccount:parameters.account];
```

Comment:
nit: Not immediately sure what's being filled in with the account. Maybe "setAccountIdentifier"?

Reply:
Addressed